sphinx/source/docs/gallery.rst

```diff
@@ -146,3 +146,12 @@ plot.
 .. bokeh-gallery:: gallery.json
 
 .. _live tutorial notebooks: https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb
+
+.. _gallery_overview:
+
+Gallery Overview
+================
+
+Below are all standalone examples listed in alphabetical order.
+
+.. bokeh-gallery-overview::
```

Comment:
I think it is this:

* create a new file `gallery_index.rst` with
   ```
   .. _gallery_index:

   Gallery Index
   =============
   ```
And then from anywhere else in the docs have a 
```
:ref:`gallery_index`
```

Alternatively this could be make and `:orphan:` page for now.

Reply:
cc @tcmetzger what do you think about a page with tabs for different indices? E.g. ideally a "global" index, but also an index just for `examples/plotting` and seperately for `examples/models`

Reply:
Or maybe separate pages for indexes would be better so that each could get RHS page navigation (just links for alphabet letters). Currently it's bit hard to navigate the very long index page. 

* `example_index.rst`
* `example_index_models.rst`
* `example_index_plotting.rst`

Then maybe later we can add `example_index_keyword` that indexes by those keywords that are currenty just ornamental.